src/main/steps/common/page.njk

```diff
@@ -80,6 +80,18 @@
   </div>
 {% endblock %}
 
+{% block footer %}
+  {{ govukFooter({
```

Comment:
this needs to be imported, I've also included but i think your changes should go in first so i believe its best to include it.